test/DynamoCoreWpfTests/WebRequestTests.cs

```diff
@@ -66,6 +67,34 @@ public void WebRequest()
             Assert.Pass("There were {0} evaluations.", ws.EvaluationCount);
         }
 
+        [Test]
+        public void DateTimeNow()
+        {
+            var openPath = Path.Combine(GetTestDirectory(ExecutingDirectory), @"core\periodic_update\datetimenow.dyn");
+            ViewModel.OpenCommand.Execute(openPath);
+
+            var ws = Model.CurrentWorkspace as HomeWorkspaceModel;
+            var dataTimeNow = ws.FirstNodeFromWorkspace<DSFunction>();
+            Assert.NotNull(dataTimeNow);
+
+            var time1 = GetPreviewValue(dataTimeNow.GUID.ToString()).ToString();
+
+            var stopWatch = new Stopwatch();
+            stopWatch.Start();
+
+            // Switch to periodic mode
+            ws.RunSettings.RunPeriod = 100;
+            ws.RunSettings.RunType = RunType.Periodic;
+
+            while (stopWatch.Elapsed.Seconds <= 1) { }
+
+            stopWatch.Stop();
+            var time2 = GetPreviewValue(dataTimeNow.GUID.ToString()).ToString();
```

Comment:
Minor. dataTimeNow.GUID.TroString() in line 80 and line 92 can be moved out and assigned to a local variable.